app/src/main/java/com/example/learingrealmandretrofit/SharedPreferencesManager.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.learingrealmandretrofit
+
+import android.content.Context
+import android.content.SharedPreferences
+import com.example.learingrealmandretrofit.objects.UserSharedPref
+
+class SharedPreferencesManager(context: Context) {
+    private var sharedPreferences: SharedPreferences = context.getSharedPreferences(context.getString(R.string.app_name), Context.MODE_PRIVATE)
+
+    companion object {
+        const val userTokenKey = "USER_TOKEN"
```

Comment:
Use namespacing, because other libraries might have their own `USER_TOKEN`, `USER_ID`, etc.
For example `com.example.learingrealmandretrofit.USER_ID`